Gruntfile.coffee

```diff
@@ -1299,45 +1303,47 @@ taskManager.add 'socksEchoIntegrationTest', [
   'jasmine_chromeapp:socksEcho'
 ]
 
+taskManager.add 'lint', [
+  'tslint'
+  'jshint'
+]
+
+taskManager.add 'unit_test', [
+  'test_lib'
+  'test_core'
+  'test_ui'
+  'test_chrome'
+]
+
 # TODO: add test_chrome once it passes reliably
 taskManager.add 'integration_test', [
   'tcpIntegrationTest'
   'socksEchoIntegrationTest'
 ]
 
-taskManager.add 'everything', [
-  'build'
-  'test'
-  'integration_test'
-]
-
-# This is the target run by Travis. Targets in here should run locally
-# and on Travis/Sauce Labs.
 taskManager.add 'test', [
-  'exec:lintFirefoxJs'
-  'test_lib'
-  'test_core'
-  'test_ui'
-  'test_chrome'
+  'unit_test'
+  'integration_test'
 ]
 
+# Builds all code, including the "dist" build, but skips
+# linging and testing which can both be annoying and slow.
```

Comment:
Nice. And I found at least running jshint from the command line on those few .js files was pretty fast, so if the grunt plugin is anything like it, this shouldn't slow the build step down much.